build.gradle

```diff
@@ -67,7 +68,7 @@ dependencies {
     // json
     implementation 'org.json:json:20201115'
 
-    implementation 'ch.qos.logback:logback-classic:1.2.3'
+    implementation 'ch.qos.logback:logback-classic'
```

Comment:
Can you explain why removing the version?

Reply:
It is not about removing the version, it is about using spring BOM and versions defined within the BOM.
As much as possible, we will be using the same set of versions as those defined in a given spring release BOM:
https://github.com/iExecBlockchainComputing/iexec-common/blob/feature/docker-rework-step-1/build.gradle#L40

Reply:
I missed that part, thanks!